.gitignore

```diff
@@ -13,3 +13,4 @@
 dist
 build
 venv*
+docker-compose.yaml
```

Comment:
It looks like the standard name has changed in the official docs: https://docs.docker.com/compose/gettingstarted/#step-2-define-services-in-a-compose-file
Feel free to change it back. I hadn't kept up with the times apparently.

Reply:
It's fine. I don't mind :smile: